build/manifest/main.go

```diff
@@ -76,17 +80,22 @@ func findManifest() (*model.Manifest, error) {
 	return &manifest, nil
 }
 
-// dumpPluginId writes the plugin id from the given manifest to standard out
-func dumpPluginId(manifest *model.Manifest) {
+// dumpPluginID writes the plugin id from the given manifest to standard out
+func dumpPluginID(manifest *model.Manifest) {
```

Comment:
Can we leave it as `dumpPluginId`? I know some style checkers complain about it but this is how we do it all over and we try to be consistent

Reply:
0/5 for this PR, but I do not this isn't just about "some style checkers" but more about writing idiomatic Golang, e.g. https://github.com/golang/go/wiki/CodeReviewComments#initialisms